prql-elixir/native/prql/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "prql"
```

Comment:
I know you asked this on discord and I said `prql` was fine. But I _think_: 
- the _rust crate_ should be called `prql-elixir`
- we should reference that from the root `Cargo.toml` so it builds with everything else
- we should also take the `publish` & `version.workspace` from https://github.com/prql/prql/blob/main/prql-js/Cargo.toml#L6-L8

Then to the extent the `elixir` package can be called something different, then we can follow the elixir convention.

But if we can't have different names between languages and `prql-elixir` is inconsistent with convention, then we can reassess.

Reply:
This rust crate only interface with elixir, generally we use the same naming conversation for the crate and elixir otp_app

Reply:
so essentially prql in this crate means, its referring to otp_app called prql. Thats the elixir package name

Reply:
OK. I guess this depends on whether the crate should be in the prql workspace in https://github.com/prql/prql/blob/main/prql-js/Cargo.toml#L6-L8, or just independent in the repo.

Having it independent in the repo is functionally the same as having it in a different repo from a packaging POV, I think.

I think either is fine. If it's independent, it's more difficult to test and release them together. But that's no great problem.

Maybe we try leaving it like this, don't add it to the workspace, and we see how that goes? (or if others have views, please comment...)

Reply:
its independent, I had to turn workspace to empty to run it.

Reply:
Sorry I was on mobile all the time for previous replies.

1. This rust crate acts as the NIF for elixir repo, its not related to the workspace project
2. Elixir Rustler handles the building and linking the NIF without the workspace(even works with a remote rust crate)
3. The standard practice is to name your NIF crate with the same name as the OTP app(We can config this, but this crate name makes sense in my opinion). Read More [here](https://hexdocs.pm/rustler/Rustler.html#module-configuration-options)
4. Since Elixir purely handles the building/testing/releasing of this crate, we can make it a part of Elixir package building process which is independent from others

Reply:
OK, that's very helpful, thanks.

We lose some cohesion, like integrated versioning and build/testing; but that's not essential. If that becomes inconvenient, we can explore whether having it part of the workspace is even possible.

Reply:
yes, as I can see all other bindings are built using the workspace. But for elixir this is not the case